doc/sphinx/source/projects_doc.rst

```diff
@@ -0,0 +1,15 @@
+*******************
+no-OS projects list
+*******************
+
+The following sections contain code documentation for ADI no-OS projects.
+
+INERTIAL MEASUREMENT UNITS
+==========================
+.. toctree::
+   :maxdepth: 1
+
+   ../projects/eval-adis1646x
```

Comment:
is `../` really needed here? wondering if we can use the same approach as the one we used for drivers.

Reply:
nope, it is not needed, I made the updat